### DISCUSSION TITLE
Validating a WZDx data feed (v2 GeoJSON format)

### DISCUSSION THREAD

**Discussion by wesleyalforddot** [usdot-jpo-ode/jpo-wzdx#124] [Show and tell]
During the process of developing a WZDx data feed your organization will want to test your prototype feed against the specification. One way to do this is to compare it to the [JSON schema developed here](https://github.com/usdot-jpo-ode/jpo-wzdx/blob/master/create-feed/schemas/wzdx_v2.0_feed.json).

To do this in an automated fashion, you can use a JSON validator [like the one found here](https://www.jsonschemavalidator.net/). You can also use the following link to go to a version that already contains the WZDx v2 GeoJSON schema: [https://www.jsonschemavalidator.net/s/d2oYmEnj](https://www.jsonschemavalidator.net/s/d2oYmEnj)

![image](https://user-images.githubusercontent.com/31485082/90661043-b90bc100-e214-11ea-8eb1-3b3676b36d2d.png)

Lastly, you can visually validate the data feed by using GeoJSON visualization tool like the one found here: [http://geojson.io/#map=2/20.0/0.0](http://geojson.io/#map=2/20.0/0.0l). This can help catch data errors like reversing latitude and longitude.

![image](https://user-images.githubusercontent.com/31485082/90661159-d8a2e980-e214-11ea-9509-040c2e977593.png)

**Comment by j-d-b** [comment]
Thanks for posting. I will add this to the `create-feed` readme in the repo as well.
